src/components/modal/modal.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
```

Comment:
Should we leave a comment in this file outlining that is was copied and why?

Reply:
Yes. Added.